src/enforcer.ts

```diff
@@ -26,30 +26,33 @@ export class Enforcer extends ManagementEnforcer {
    * initWithFile initializes an enforcer with a model file and a policy file.
    * @param modelPath model file path
    * @param policyPath policy file path
+   * @param lazyLoad lazyLoad whether to load policy at initial time
    */
-  public async initWithFile(modelPath: string, policyPath: string): Promise<void> {
+  public async initWithFile(modelPath: string, policyPath: string, lazyLoad = false): Promise<void> {
```

Comment:
Why no param type like boolean? Is this OK for TS?

Reply:
no problem.